ocaml/libs/http-lib/dune

```diff
@@ -133,14 +133,10 @@
   )
 )
 
-(rule
-  (alias runtest)
+(cram
   (package xapi)
   (deps
     test_client.exe
     test_server.exe
-    client_server_test.sh
```

Comment:
how does it find the new test_client_server.t?

Reply:
The `cram` atom processes all the `.t` files in the directory